.github/knip.ts

```diff
@@ -3,10 +3,10 @@ import type { KnipConfig } from "knip";
 const config: KnipConfig = {
   entry: ["build/index.ts"],
   project: ["src/**/*.ts"],
-  ignore: ["src/types/config.ts", "**/__mocks__/**", "**/__fixtures__/**", "src/worker.ts"],
+  ignore: ["src/types/config.ts", "**/__mocks__/**", "**/__fixtures__/**", "src/worker.ts", "src/types/plugin-input.ts", "src/main.ts", "src/plugin.ts"],
```

Comment:
Why are these ignored?

Reply:
https://github.com/ubiquibot/command-start-stop/pull/10#issuecomment-2228807316 this knip comment above

https://github.com/ubiquibot/command-start-stop/actions/runs/10049073859/job/27774413984?pr=10

> Unused item in ignoreDependencies: @actions/core
> Unused item in ignoreDependencies: @actions/github

These are the only ignored items that are not hitting apparently

Reply:
Are these actually used?
Is knip actually running on this branch during the tests? I suspect it runs on the `development` branch to post this.

Reply:
the workflow runs on `pull_request_target` and the actions tab shows it pitted against my fork/branch so I believe it's running against this PR.

Yeah everything mentioned in the knip comment above is being used, perhaps not the `main.ts` and `plugin.ts`

Reply:
If the `main` is not recognized possibly some files will be considered as unused because they are reference through the main. But in other projects we never needed to ignore these files.

Reply:
resolved